specs/api_cron_flow.md

```diff
@@ -0,0 +1,82 @@
+# Automating the API Update Flow with Cron
+
+## Problem Statement
+
+Some of the extern APIs that we use may not allow us to process all of
+our data records in a single batch. If we send too many requests to
+the API in a short time, our access may be throttled or suspended.
+
+## Background and Significance
+
+
+There are currently two APIs that we use to enhance the data sets from
+the SBA.
```

Comment:
Just FYI PR #25 now adds a third external API we'll use: getting lat/long from Google Places.

Reply:
Yeah, I saw that. We'll need to merge them in. I can make my document more generic.